src/Pages/AccountPage/Login.js

```diff
@@ -20,7 +51,7 @@ function Login() {
               </span>
               <ReSettingPw>비밀번호 재설정</ReSettingPw>
             </IdSelect>
-            <DefaultLogin>로그인</DefaultLogin>
+            <DefaultLogin onClick={() => loginFetch()}>로그인</DefaultLogin>
```

Comment:
```suggestion
            <DefaultLogin onClick={loginFetch}>로그인</DefaultLogin>
```
즉시실행 함수로 만들어주지 않아도 될 것 같습니다. 다음 PR에서 수정해주세요.